libs/azure-storage/README.md

```diff
@@ -1,9 +1,184 @@
 # langchain-azure-storage
 
-This package contains the LangChain integrations for Azure Storage. This project is a work-in-progress and currently does not contain any features.
+This package contains the LangChain integrations for [Azure Storage](https://learn.microsoft.com/en-us/azure/storage/common/storage-introduction). Currently, it includes:
+- [Document loader support for Azure Blob Storage](#azure-blob-storage-document-loader-usage)
+
+> [!NOTE]
+> This package is in Public Preview. For more information, see [Supplemental Terms of Use for Microsoft Azure Previews](https://azure.microsoft.com/support/legal/preview-supplemental-terms/).
 
 ## Installation
 
 ```bash
 pip install -U langchain-azure-storage
 ```
+
+## Configuration
+`langchain-azure-storage` should work without any explicit credential configuration.
+
+The `langchain-azure-storage` interface defaults to [`DefaultAzureCredential`](https://learn.microsoft.com/en-us/azure/developer/python/sdk/authentication/credential-chains?tabs=dac#defaultazurecredential-overview)
+for credentials which automatically retrieves [Microsoft Entra ID tokens](https://learn.microsoft.com/en-us/azure/storage/blobs/authorize-access-azure-active-directory) based on
+your current environment. For more information on using credentials with
+`langchain-azure-storage`, see the [override default credentials](#override-default-credentials) section.
+
+## Azure Blob Storage Document Loader Usage
+[Document Loaders](https://python.langchain.com/docs/integrations/document_loaders/) are used to load data from many sources (e.g., cloud storage, web pages, etc.) and turn them into [LangChain Documents](https://python.langchain.com/api_reference/core/documents/langchain_core.documents.base.Document.html), which can then be used in AI applications. This package introduces the `AzureBlobStorageLoader` which downloads blob content from Azure Blob Storage and parses it as UTF-8 by default. Additionally, [parsing customization](#customizing-blob-content-parsing) is also available to handle content of various file types.  
```

Comment:
Instead of using the word `introduce`, let's switch it to `This package offers the`... Mainly if this were a blog post, introduce would be more appropriate but the readme is more geared toward reference/usage documentation.